src/Makefile.am

```diff
@@ -24,8 +25,9 @@ dist_cache_DATA = .keep
 dist_varlib_DATA = .keep
 dist_registry_DATA = .keep
 dist_log_DATA = .keep
+plugins_PROGRAMS =
 if !MACOS
```

Comment:
we already detect the MACOS at autoconf, so define AM_CONDITIONAL of ENABLE_PLUGIN_APPS to true/false based on MACOS but in autoconf.